lib/recurly/custom_field_definition_list.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+class Recurly_CustomFieldDefinitionList extends Recurly_Pager
+{
+  public static function get($params = null, $client = null) {
+    $uri = self::_uriWithParams(Recurly_Client::PATH_CUSTOM_FIELD_DEFINITIONS, $params);
+    return new self($uri, $client);
+  }
+
+  public static function getByRelatedType($related_type, $client = null) {
+    $params['related_type'] = $related_type;
+    return self::get($params, $client);
+  }
```

Comment:
**question**: What prompted two public functions as opposed to just passing `related_type` as a key/value of `$params` to `get`?  

My concern is, if this isn't a foregoing pattern of the other classes in PHP, then we're forever adding new `getByX(...)` variations going forward.

Reply:
I saw in [Accounts](https://github.com/recurly/recurly-client-php/blob/v2/lib/recurly/account_list.php#L5) where there are methods for getActive, getClosed… so I based on there. But as it's only a filter I'm ok in keeping only the `get`.